### DISCUSSION TITLE
Option to Censor Farm ID and SFL Balance

### DISCUSSION THREAD

**Discussion by Polysemouse** [sunflower-land/sunflower-land#1182] [Ideas]
**Issue**
Currently the Farm ID and SFL balance is shown all the time when players are playing Sunflower Land.  Some users need privacy when playing their farm, or sharing their progress with screenshots.

**Proposal**
 A toggle functionality can be added next to the farm ID UI at the bottom left of the screen. 
It will be similar to how hiding the balance of account in crypto changes work, such as Binance, Coinbase or Crypto.com.  This allows greater privacy for players when playing their farms around other people.  It also allow them to share screenshots easier without having to cross out their farm ID and SFL balance every time.

**Comment by alphacruze** [comment]
The game is on the Blockchain, meaning that even if the person does not go to the farm, said information can be found either way, the game was not built to hide information but to make it accessible hence on the blockchain, after all Blockchains are PUBLIC Ledgers where the information is stored in the form of Transactions

If I recall correctly the information shown on a farm is only the information from the last time someone synched, meaning all the information is publicly available on the blockchain, just being visually represented on the farms

**Comment by adamhannigan** [comment]
@Polysemouse Thanks for raising you concern.

Displaying the Farm ID has a secondary purpose. It helps us catch botters and multiaccounters who share screenshots/videos of their farms 😅 

There are a surprising amount of people who get caught this way so to protect the community we will leave this feature in for the short term.

With Land Expansion we will investigate other methods for getting a hidden ID on the farm 😄

**Reply by Polysemouse** [reply]
I didn't think they will forget to hide their farm ID before sending their screenshots 😅 guess we can keep the farm ID on screen for now.

In the future, this feature (hiding the ID and SFL balance) could be an unlockable from some achievements or crafting recipe
